src/components/collection-progress-group/components/collection-progress-status-section/index.tsx

```diff
@@ -1,11 +1,25 @@
-import { ReactNode } from 'react'
+import { ReactElement } from 'react'
 import { IconCollections16 } from '@hashicorp/flight-icons/svg-react/collections-16'
 import { IconCheckCircleFill16 } from '@hashicorp/flight-icons/svg-react/check-circle-fill-16'
 import ProgressBar from 'components/progress-bar'
+import { CollectionProgressStatusSectionProps } from './types'
 import s from './collection-progress-status-section.module.css'
 
 /**
  * Renders collection progress status, in a nice little box.
+ */
+function CollectionProgressStatusSection(
```

Comment:
I've refactored this slightly to expand the layers of composition, as `CollectionCard` needs the same progress elements, but without the border and padding used in the sidebar and in the collection meta sections.